src/witan/gwyn/gwyn.clj

```diff
@@ -67,10 +70,26 @@
 
 (defn collect-property [json-entry]
   (hash-map :address (get json-entry "vicinity")
-            :type (get json-entry "types")
+            :type (set (get json-entry "types"))
             :id (get json-entry "id")
             :name (get json-entry "name")))
 
+(def unwanted-properties
+  (edn/read-string
+   (slurp (io/file "data/unwanted_property_types.edn"))))
```

Comment:
I thought it'd be cleaner in an external file, but I can just add it to the utils ns and import it.